temporalcli/commands.activity_test.go

```diff
@@ -231,6 +231,37 @@ func (s *SharedServerSuite) TestActivityUnPause_Failed() {
 	s.Error(res.Err)
 }
 
+func (s *SharedServerSuite) TestActivityReset() {
+	run := s.waitActivityStarted()
+	wid := run.GetID()
+	aid := "dev-activity-id"
+	identity := "MyIdentity"
+
+	res := s.Execute(
+		"activity", "reset",
+		"--activity-id", aid,
+		"--workflow-id", wid,
+		"--run-id", run.GetRunID(),
+		"--identity", identity,
+		"--address", s.Address(),
+	)
+
+	s.NoError(res.Err)
```

Comment:
Didn't mention it on the last review but I should have. Ideally these tests should confirm the invoked action actually occurs.